src/role-helpers.js

```diff
@@ -138,23 +139,30 @@ function getRoles(container, {hidden = false} = {}) {
     }, {})
 }
 
-function prettyRoles(dom, {hidden}) {
+function prettyRoles(dom, {hidden, includeName}) {
   const roles = getRoles(dom, {hidden})
 
   return Object.entries(roles)
     .map(([role, elements]) => {
       const delimiterBar = '-'.repeat(50)
       const elementsString = elements
-        .map(el => prettyDOM(el.cloneNode(false)))
+        .map(
+          el =>
+            `${
+              includeName === true
+                ? `Name "${computeAccessibleName(el)}":\n`
+                : ''
+            }${prettyDOM(el.cloneNode(false))}`,
+        )
```

Comment:
Or maybe it would be better to just do this:

```suggestion
        .map(el => {
          const nameString =
            includeName === true ? `Name "${computeAccessibleName(el)}":\n` : ''
          const domString = prettyDOM(el.cloneNode(false))
          return `${nameString}${domString}`
        })
```